src/workerd/jsg/setup.c++

```diff
@@ -145,6 +145,11 @@ void IsolateBase::terminateExecution() const {
   ptr->TerminateExecution();
 }
 
+void IsolateBase::requestGcForTesting() const {
+  ptr->RequestGarbageCollectionForTesting(
+    v8::Isolate::GarbageCollectionType::kFullGarbageCollection);
```

Comment:
Non-blocking nit: It might be worthwhile parameterizing `requestGcForTesting(...)` to allow requesting full or minor GC. I think most of the time we'll want full when we use this so defaulting to that makes sense. We can always add this later tho

Reply:
Considering that this is only going to be used for testing I think that we can add it later if we need it. I don't know if there will be many other uses of this, maybe if someone was doing a ew-test bin.